x/sequencer/types/msg_create_sequencer.go

```diff
@@ -104,6 +118,18 @@ func (msg *MsgCreateSequencer) ValidateBasic() error {
 		return errorsmod.Wrapf(ErrInvalidCoins, "invalid bond amount: %s", msg.Bond.String())
 	}
 
+	if msg.RewardAddr != "" {
+		_, err = sdk.AccAddressFromBech32(msg.RewardAddr)
+		if err != nil {
+			return errorsmod.Wrap(errors.Join(gerrc.ErrInvalidArgument, err), "get reward addr from bech32")
+		}
+	}
+
+	err = ValidateWhitelistedRelayers(msg.WhitelistedRelayers)
```

Comment:
I think we should limit the list of relayer addresses. 10 should be more than enough.